src/main/java/mate/academy/books/BooksApplication.java

```diff
@@ -1,13 +1,32 @@
 package mate.academy.books;
 
+import java.util.List;
+import mate.academy.books.model.Book;
+import mate.academy.books.repository.BookRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.CommandLineRunner;
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.context.annotation.Bean;
 
 @SpringBootApplication
 public class BooksApplication {
+    @Autowired
+    private BookRepository bookRepository;
 
     public static void main(String[] args) {
         SpringApplication.run(BooksApplication.class, args);
     }
 
+    @Bean
+    public CommandLineRunner commandLineRunner() {
+        return args -> {
+            Book book = new Book();
+            book.setTitle("Book 1");
+
+            bookRepository.save(book);
```

Comment:
You should call BookService methods, I mentioned this here https://github.com/bkostanetskyi/Books/pull/1#discussion_r1763563436

Implement this comment and run you application locally to get the result